docs/File-Formats.md

```diff
@@ -1296,18 +1296,21 @@ pivot_threshold_value: 0.1
 value_sort_order: ASC
 ```
 
-## Note on `value_sort_order`
+#### Note on `value_sort_order`
 When values are sorted based on the `value_sort_order`, data points at the start of the sequence are considered more significant than data points at the end. This concept is used by the oncoprint when aggregating treatment response data from multiple samples from a single patient. When `value_sort_order` is `ASC` the sample with the smallest response value will be shown for that patient. When `value_sort_order` is `DESC` the sample with the largest response value will be shown for that patient.
 
-### Treatment response IC50 data file
-The data file will be a simple tab separated format, similar to the expression data file: each sample is a column, each tested treatment (a compound or combination thereof) is a row, each cell contains treatment IC50 values for that sample x treatment combination.
+#### Note on `GENERIC_ASSAY` genetic_alteration_type and `LIMIT-VALUE` datatype
+Treatment response data is registered to be of the `GENERIC_ASSAY` genetic_alteration_type and data type `LIMIT-VALUE`. This alteration and data type is intended to be used for any numerical data set with similar structure (entities measured in samples).
+
+### Treatment response data file
+The data file will be a simple tab separated format, similar to the expression data file: each sample is a column, each tested treatment (a compound or combination thereof) is a row, each cell contains treatment response values for that treatment x sample combination.
 
-The first column must be named `treatment_id` and contains unique identifiers for the treatments. The `treatment_id` column can be followed by optional unique `name`, `description` and `url` columns. The other columns are sample columns; an additional column for each sample in the dataset using the sample id as the column header.
+The first column must be named `stable_id` and contains unique identifiers for the treatments. The `stable_id` column can be followed by optional `META:name`, `META:description` and `META:url` columns. The other columns are sample columns; an additional column for each sample in the dataset using the sample id as the column header.
```

Comment:
see my previous comment about `stable_id`

Reply:
All occurences of `stable_id` were renamed to `entity_stable_id`.